uvicorn/workers.py

```diff
@@ -62,7 +63,10 @@ def init_process(self):
         super(UvicornWorker, self).init_process()
 
     def init_signals(self):
-        pass
+        # Reset signals so Gunicorn doesn't swallow subprocess return codes
+        # other signals are set up by Server.install_signal_handlers()
```

Comment:
I'm going ahead and adding a reference to the issue here too :)

```suggestion
        # other signals are set up by Server.install_signal_handlers()
        # See: https://github.com/encode/uvicorn/issues/894
```